api/src/pcapi/routes/backoffice/offerers/validation_repository.py

```diff
@@ -280,6 +274,27 @@ def list_offerers_to_be_validated(
         offerers_models.Offerer.id,
     )
 
+    # after _apply_query_filters so that outerjoin below is after union (not required for filters),
+    # otherwise sqlalchemy raises "cartesian product" error in case of free text search.
+    query = query.outerjoin(
+        offerers_models.IndividualOffererSubscription, offerers_models.Offerer.individualSubscription
+    ).options(
+        sa.orm.contains_eager(offerers_models.Offerer.individualSubscription).load_only(
+            offerers_models.IndividualOffererSubscription.isEmailSent,
+            offerers_models.IndividualOffererSubscription.isCriminalRecordReceived,
+            offerers_models.IndividualOffererSubscription.isCertificateReceived,
+        ),
+    )
+
+    if ae_documents_received:
+        query = query.filter(
+            offerers_models.IndividualOffererSubscription.isEmailSent.is_(True),
+            sa.and_(
+                offerers_models.IndividualOffererSubscription.isCriminalRecordReceived.is_(True),
+                offerers_models.IndividualOffererSubscription.isCertificateReceived.is_(True),
```

Comment:
[question] Ici on filtre sur les valeurs `True` en base de données (quand l'utilisateur sélectionne "oui", je suppose). Mais je ne vois pas le code qui filtrerait sur `False` quand l'utilisateur sélectionne "non". Est-ce voulu ?

Je ne sais pas vraiment quel mécanisme transforme les "yes" et "no" (transmis par le navigateur) en booléen. Je loupe peut-être quelque chose...

Reply:
La valeur demandée est `ae_documents_received`, et le filtrage est fait juste en-dessous par `in_(ae_documents_received)`.

La condition à vérifier `(isCriminalRecordReceived = true) and (isCertificateReceived = true)`
On vérifie que cette expression est `in_(ae_documents_received)`.

Reply:
Oups, je n'ai répondu qu'à une partie, a priori les "yes" et "no" sont bien transformés en booléens par `sqlalchemy.utils.langhelpers.asbool` : 
https://github.com/sqlalchemy/sqlalchemy/blob/rel_1_4/lib/sqlalchemy/util/langhelpers.py#L1248

Reply:
J'avais mal lu et complètement loupé `.in_(ae_documents_received)`. Et mal interprété le `if ae_documents_received`... Je vais retourner dans mon garage ! :)